### DISCUSSION TITLE
Source Mac Address of LLDP Packet

### DISCUSSION THREAD

**Discussion by jcubeaudi0** [lldpd/lldpd#515] [Q&A]
Hi!

Is there a way to configure the source mac address of the LLDP tx packet ? 

Thank you!

jcube

**Comment by vincentbernat** [comment]
Only for active/backup bonds with `configure system bond-slave-src-mac-type`.

**Reply by jcubeaudi0** [reply]
Thanks Vincent. I saw that option but it does not do what I would like to do.
